### ISSUE TITLE
Move to Zenodo

### ISSUE THREAD

**Issue by dustalov** [nlpub/russe#2]
Recently, [Zenodo](https://zenodo.org) removed the 2 GB file limit, so we can upload our models and datasets there. Now the limit is 50 GB, which is currently not a problem for us. Each dataset receives a DOI. The files are hosted by CERN.

What do you think about such a transition? Indeed, the links to the mirrors will not be removed.

**Comment by alexanderpanchenko** [comment]
looks very interesting: 50g limit is per file or for all files?

2016-10-30 17:25 GMT+01:00 Dmitry Ustalov notifications@github.com:

**Comment by dustalov** [comment]
They claim the 50 GB per dataset limit. AFAIR, 50 GB for corpus, another 50 GB for word2vec, etc.

**Comment by alexanderpanchenko** [comment]
and no limits overall?

2016-10-30 18:07 GMT+01:00 Dmitry Ustalov notifications@github.com:

**Comment by dustalov** [comment]
I see no other limits. Well, at least one file should be included into dataset.

**Comment by alexanderpanchenko** [comment]
i am trying to upload the RDT now (1.8Gb)

however I will need another machine as I do not have enough space to upload
other datasets :-)

2016-10-30 18:17 GMT+01:00 Dmitry Ustalov notifications@github.com:

**Comment by dustalov** [comment]
I can upload other datasets next week.

**Comment by alexanderpanchenko** [comment]
probably i need to add myself as this is done from my account.

2016-10-30 18:20 GMT+01:00 Dmitry Ustalov notifications@github.com:

**Comment by dustalov** [comment]
The primary issue is that I can not find the button for adding the collaborators who can edit the dataset metadata, if necessary.

**Comment by alexanderpanchenko** [comment]
https://zenodo.org/record/163857#.WBYzvZMrKHo

2016-10-30 18:32 GMT+01:00 Dmitry Ustalov notifications@github.com:

**Comment by alexanderpanchenko** [comment]
can you update the web page adding this link?

2016-10-30 18:32 GMT+01:00 Dmitry Ustalov notifications@github.com:

**Comment by dustalov** [comment]
Sure, but not right now. Only tomorrow. BTW, I believe the upload type should be dataset, not publication, since we do not include the paper.

**Comment by dustalov** [comment]
@alexanderpanchenko, BTW, I believe the upload type should be dataset, not publication, since we do not include the paper.

**Comment by dustalov** [comment]
Something like this: http://russe.nlpub.ru/downloads/#rdt-russian-distributional-thesaurus.

**Comment by alexanderpanchenko** [comment]
yes, but i put all datasets, not only RDT there (except two large uploads
yet)
so i would put the link on the top of the page

2016-10-31 13:11 GMT+01:00 Dmitry Ustalov notifications@github.com:

**Comment by dustalov** [comment]
I plan to remake the RUSSE website. Just consider this as a pilot.

**Comment by alexanderpanchenko** [comment]
ok

2016-10-31 14:22 GMT+01:00 Dmitry Ustalov notifications@github.com:
